downloader/src/main/kotlin/Downloader.kt

```diff
@@ -384,6 +385,11 @@ class Downloader {
             DownloadResult(startTime, outputDirectory, sourceArtifact = pomArtifact)
         } catch (e: FileNotFoundException) {
             throw DownloadException("Failed to download the Maven POM for '${target.id}'.")
+        } catch (e: MalformedURLException) {
+            // Many packages do have an empty string as binary artifact URL.
```

Comment:
Nit: This line is something *we* experience from looking at our internal analyzer results, but it should not go as a general comment here.

Reply:
Likewise, the similar sentence should be omitted the commit message.